src/schema/v2/image/ScaledImage.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLFloat,
+  GraphQLNonNull,
+} from "graphql"
+import { scale } from "proportional-scale"
+import { ResolverContext } from "types/graphql"
+
+export type TScaledImage = ReturnType<typeof scale> & {
+  src: string
+  srcSet: string
+}
+
+export const ScaledImage = new GraphQLObjectType<TScaledImage, ResolverContext>(
```

Comment:
Ah this should actually be ScaledImageType, not ScaledImage, which would let me avoid the TScaledImage naming.